app/assets/stylesheets/v3/layout_components/buttons/buttons.scss

```diff
@@ -111,6 +111,44 @@
     }
   }
 
+  &--ghost {
+    @extend .button;
+    display: flex;
+    text-align: left;
+    align-items: center;
+    background-color: $white;
+    border: 3px solid $blue-main-brand;
+    min-height: 4.5rem;
+    padding: $base-padding/2 $base-padding*3 $base-padding/2 $base-padding;
+    color: $blue-main-brand;
+    vertical-align: middle;
+    line-height: 1.65rem;
+    position: relative;
```

Comment:
why do we need this?

Reply:
Oh no!! Maybe I won't be able to get away with flexbox after all. I'll look into a different solution.

Reply:
We need this for the ::after position being set to absolute. Without explicitly setting the position of the parent, the arrows won't sit properly in each button.